fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/impl/PropertiesRdfContext.java

```diff
@@ -66,7 +68,14 @@ public PropertiesRdfContext(final FedoraResource resource,
     private Iterator<Triple> triplesFromProperties(final FedoraResource n)
         throws RepositoryException {
         LOGGER.trace("Creating triples for node: {}", n);
-        final Iterator<Property> allProperties = n.getNode().getProperties();
+        final Iterator<Property> allProperties;
+        if (n instanceof FedoraBinary) {
```

Comment:
I see, so the logic should be first get properties from n.getNode().getProperties() and then if the node is a binary, append description.getNode().getProperties(). 
Can you tell me how to append allProperties?